src/renderer/containers/widgets/channels/ChannelMessages.tsx

```diff
@@ -49,6 +49,10 @@ export const ChannelMessages = ({ contentRect }) => {
   const displayableMessageLimit = useSelector(
     channelSelectors.displayableMessageLimit
   )
+
+  const isDisplayableMessageLimit = (displayableMessageLimit: number | undefined): displayableMessageLimit is number => {
+    return typeof displayableMessageLimit === 'number'
```

Comment:
you don't need explicit guard function for `typeof ... === 'number'`

simple
```
if (typeof displayableMessageLimit === 'number') {
...
}
```
has same effect as implicit type guard